database/factories/CelebrityFactory.php

```diff
@@ -26,6 +26,16 @@ public function definition()
             'dob' => $this->faker->date,
             'nationality' => $this->faker->country,
             'bio' => $this->faker->text,
+            'poster' => $this->randomposter(),
         ];
     }
+
+    public function randomposter()
+    {
+        $posters = [
+            'https://upload.wikimedia.org/wikipedia/commons/4/46/Leonardo_Dicaprio_Cannes_2019.jpg'
```

Comment:
Probably try to look for more pictures to put in here, that way it will look much more dynamic ;D